lib/harmony/harmonydb/sql/20231103.sql

```diff
@@ -0,0 +1,22 @@
+create table message_sends
+(
+    from_key     text               not null,
+    nonce        bigint             not null,
+    to_addr      text               not null,
+    signed_data  bytea              not null,
+    signed_json  jsonb              not null,
+    signed_cid   text               not null,
+    send_time    date default CURRENT_TIMESTAMP,
```

Comment:
date is day resolution afaict, want better

Reply:
Consider TIMESTAMP (1 second resolution)